packages/components/socialFeed/NewsFeed/MapModal.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { Suspense, lazy, useState } from "react";
+import { Platform, View, ViewStyle } from "react-native";
+
+import { AddressSearch } from "./MapComponent/AddressSearch";
+import { MapDescriptionInput } from "./MapComponent/MapDescriptionInput";
+import { MapHeader } from "./MapComponent/MapHeader";
+import {
+  neutral17,
+  primaryColor,
+  transparentColor,
+} from "../../../utils/style/colors";
+import { layout } from "../../../utils/style/layout";
+import { PrimaryButton } from "../../buttons/PrimaryButton";
+import { SecondaryButtonOutline } from "../../buttons/SecondaryButtonOutline";
+import { SpacerColumn, SpacerRow } from "../../spacer";
+
+import { ModalWithoutHeader } from "@/components/modals/ModalWithoutHeader";
+
+const MapView = Platform.select({
+  native: () => lazy(() => import("./MapComponent/Map.native")),
+  web: () => lazy(() => import("./MapComponent/Map.web")),
```

Comment:
maybe here is the issue with the WebView @clegirar ?

Reply:
for native and web have different map package, so we have to load like seprate component like this,

Reply:
yes @omniwired that's the issue

Reply:
I recall a discussion about the "lazy" function not working in the Metro or behaving differently than in Webpack. Can you please investigate whether removing this function resolves the issue of "WebView" not being available on the web? This is just a suggestion, but it might be worth trying.

Reply:
```useIsMobile``` is define through device width so we don't use this

ex. anyone open site in mobile we have to show webview, so that not handle by ```useIsMobile```

Reply:
Oh yeah but you can just make a condition `Platform.OS === 'mobile'`, then import call mobile component else web component, is it not better ?

Reply:
both are same @clegirar

Reply:
@clegirar now import statement has beed changed.

https://reactnative.dev/docs/platform-specific-code#native-specific-extensions-ie-sharing-code-with-nodejs-and-web